src/Ot2Rec/aretomo.py

```diff
@@ -336,9 +339,17 @@ def update_yaml(args):
 
         # Set output mrc
         output_lookup = {0: "_ali.mrc", 2: "_rec.mrc"}
+        # out_file_list = [
+        #     (f"{os.path.splitext(file)[0]}"
+        #      f"{output_lookup[args['aretomo_mode']]}") for file in st_file_list]
         out_file_list = [
-            (f"{os.path.splitext(file)[0]}"
-             f"{output_lookup[args['aretomo_mode']]}") for file in st_file_list]
+            (f"{aretomo_params.params['System']['output_path']}/"
+             f"{os.path.splitext(os.path.basename(file))[0]}/"
```

Comment:
I'm guessing this does <output_path>/<TS_number>/<TS_number>_ali.mrc ?

Reply:
Yes it does - the first one without the middle <TS_number>/ would put everything in the aretomo/ folder